sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuRowToColumnarExec.scala

```diff
@@ -806,6 +817,8 @@ case class GpuRowToColumnarExec(child: SparkPlan, goal: CoalesceSizeGoal)
   }
 
   override lazy val additionalMetrics: Map[String, GpuMetric] = Map(
+    SEMAPHORE_WAIT_TIME -> createNanoTimingMetric(DEBUG_LEVEL, DESCRIPTION_SEMAPHORE_WAIT_TIME),
+    GPU_OP_TIME -> createNanoTimingMetric(MODERATE_LEVEL, DESCRIPTION_GPU_OP_TIME),
     TOTAL_TIME -> createNanoTimingMetric(MODERATE_LEVEL, DESCRIPTION_TOTAL_TIME),
```

Comment:
I am tempted to drop `TOTAL_TIME` to `DEBUG_LEVEL`, but I don't know how you want to use it with benchmarks/etc so I don't know if that is a good idea or not.

Reply:
What I would really like is OP_TIME. I should be able to calculate that as TOTAL_TIME - SEMAPHORE_WAIT_TIME - GPU_OPTIME. I'll look at that next.

Reply:
Wouldn't that just be measuring mostly the time spent in earlier child execs within the stage?  TOTAL_TIME includes time spent fetching inputs from child iterators.  It wouldn't be very op-specific, and thus OP_TIME would be an odd name for it.

Reply:
Yes, that's a good point. I would need to subtract the cost of the fetches too, but that might be expensive because this is a row-based iterator. I'll take a look and see what the options are.

Reply:
This is converting rows on the CPU to columns on the GPU.  There is close to no processing on the CPU beyond fetching data from upstream and putting it into a buffer. If we try to measure the amount of time it takes to convert from UnsafeRow to CudfUnsafeRow, or to just put it into the arrow format in a buffer (depending on the code path we take), we are likely going to spend more time measuring than actually doing the conversion. Unless knowing that number is critically important we would propose that we just lump it all together.

Reply:
Ok, I went ahead and changed the level of TOTAL_TIME to DEBUG_LEVEL.